lib/src/test/java/io/ably/lib/test/rest/HttpTest.java

```diff
@@ -147,6 +163,118 @@ public Http setUrlArgumentStack(List<String> urlArgumentStack) {
 	}
 
 	/**
+	 * Validates that fallbacks are disabled when ClientOpitons#fallbackHosts are set to null and the only host used is Defaults#HOST_REST
```

Comment:
Fixed by setting fallbackHosts to empty rather than null. The library now treats the two differently: empty means override with an empty fallback list, null means don't override.